src/composables/graph/useGraphNodeManager.ts

```
@@ -287,7 +283,59 @@ export function extractVueNodeData(node: LGraphNode): VueNodeData {
+        if (resolved) {
+          const subgraphId = node.subgraph.id
+          const sharedEnhancements = getSharedWidgetEnhancements(
+            node,
+            resolved.widget,
+            nodeId
+          )
+          const callback = (v: unknown) => {
+            const value = normalizeWidgetValue(v)
+            resolved.widget.value = value ?? undefined
+            resolved.widget.callback?.(value, app.canvas, resolved.node)
+            resolved.node.widgets?.forEach((w) => w.triggerDraw?.())
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)